### DISCUSSION TITLE
Swagger blank page

### DISCUSSION THREAD

**Discussion by liodnik** [simpleidserver/SimpleIdServer#469] [General]
It is an empty page when I try to load Swagger page. It started to happen after major update of **SimpleIdServer**. There is an error in console:
![image](https://user-images.githubusercontent.com/10138979/228234547-e7348a53-ddef-4e2f-a964-7ac35b89114d.png)

**Comment by liodnik** [comment]
I noticed there is some problem with **Routings** in **Middleware**.

**Comment by simpleidserver** [comment]
Hello,

Did-you notice an issue in the SCIM project or inside the IdServer project?

**Reply by liodnik** [reply]
Hi. I only notice it started happen after updating to major version (3.0.0, then 3.0.2) of SimpleIdServer (Scim, Domains, Parser). It can be fixed on our side with app.UseRouting() but it will be bad solution because we use .NET 6 and it should happen automatically.

**Reply by simpleidserver** [reply]
Thank you for the information. I create a bug ticket #473 in order to fix this issue. 
It will be fixed in the next release :)

**Reply by liodnik** [reply]
Great, thank you very much!
